### PR TITLE
Added three new wikis (NinWiki, RammWiki, & When They Cry)

### PR DESCRIPTION
**Pull request by DrWhoFan13**
Wikis Added:

- NinWiki from Nine Inch Nails Fandom Wiki (They've been around since 1988! How does the fandom only have _24_ articles?!)
- RammWiki from Rammstein Fandom Wiki (RammWiki has more stringent requirements for an account than the average wiki, namely you need to contact them to get an account)
- When They Cry Wiki from When They Cry Fandom

### CODE REVIEW

File: data/sitesEN.json [hunk 1]
```diff
@@ -3594,6 +3594,24 @@
     "destination_icon": "nethack.png",
     "destination_main_page": "Main_Page",
     "destination_search_path": "/index.php"
+  },
+    {
+    "id": "en-NineInchNails",
```
Comment (KevinPayravi) [inline data/sitesEN.json:3599]: IDs should be all lowercase

File: data/sitesEN.json [hunk 2]
```diff
@@ -4129,6 +4147,24 @@
       "official",
       "miraheze"
     ]
+  },
+    {
+    "id": "en-ramm",
```
Comment (KevinPayravi) [inline data/sitesEN.json:4152]: Let's make the ID the full name (`en-rammstein`)

File: data/sitesEN.json [hunk 3]
```diff
@@ -5802,6 +5838,27 @@
     "destination_icon": "lexicanum.png",
     "destination_main_page": "Main_Page",
     "destination_search_path": "/mediawiki/index.php"
+  },
+    {
+    "id": "en-whentheycry",
+    "origins_label": "When They Cry Fandom Wiki",
+    "origins": [
+      {
+        "origin": " Higurashi no Naku Koro Ni Fandom Wiki",
```
Comment (KevinPayravi) [inline data/sitesEN.json:5847]: There's an extra space at the start of the name

**Review by KevinPayravi — CHANGES_REQUESTED** [review]
Thank you! Requesting a few small fixes

**Comment by DrWhoFan13** [comment]
Errors have been fixed, just two things to note:

1. NinWiki, for some reason, doesn't have a custom favicon
2. This has the pull requests that [my other PR](https://github.com/KevinPayravi/indie-wiki-buddy/pull/616) has

**Review by KevinPayravi — APPROVED** [review]

**Pull request merged**